doc/users/next_whats_new/2020-05-28-extend_kwarg_to_BoundaryNorm.rst

```diff
@@ -0,0 +1,48 @@
+New "extend" keyword to colors.BoundaryNorm
+-------------------------------------------
+
+`~.colors.BoundaryNorm` now has an ``extend`` kwarg,
+analogous to ``extend`` in ~.axes._axes.Axes.contourf`. When set to
+'both', 'min', or 'max', it interpolates such that the corresponding
```

Comment:
interpolates what?

Reply:
Completely reworded now: 
```
`~.colors.BoundaryNorm` now has an ``extend`` keyword argument, analogous to
``extend`` in ~.axes._axes.Axes.contourf`. When set to 'both', 'min', or 'max',
it maps the corresponding out-of-range values to `~.colors.Colormap`
lookup-table indices near the appropriate ends of their range so that the
colors for out-of range values are adjacent to, but distinct from, their
in-range neighbors.  The colorbar inherits the ``extend`` argument from the
norm, so with ``extend='both'``, for example, the colorbar will have triangular
extensions for out-of-range values with colors that differ from adjacent
colors.
```